docs/dapp/tools/localnet.md

```diff
@@ -0,0 +1,118 @@
+# Local Development
+
+For convenient development and testing of your dApps the Oasis team prepared
+both [ghcr.io/oasisprotocol/sapphire-localnet][sapphire-localnet] and
+[ghcr.io/oasisprotocol/emerald-localnet][emerald-localnet] Docker images.
+They will bring you a complete Oasis network stack to your desktop. The
+Localnet Sapphire instance **mimics confidential transactions**, but it does
+not run in a trusted execution environment nor does it require Intel's SGX on
+your computer. The network is isolated from the Mainnet or Testnet and consists
+of a:
+
+- single Oasis validator node with 1-second block time and 30-second epoch,
+- single Oasis client node,
+- single compute node running Oasis Sapphire,
+- single key manager node,
+- PostgreSQL instance,
+- Oasis Web3 gateway with transaction indexer and enabled Oasis RPCs,
+- helper script which populates the account(s) for you.
+
+*Note: You will need at least 16GB of memory.*
```

Comment:
Let's use admonition. Also, looking at terminology we use [here](https://docs.oasis.io/node/run-your-node/prerequisites/hardware-recommendations#memory). Something like this?

```suggestion
:::note Hardware requirements

You will need at least 16GB of RAM to spin up the docker image apart from the OS on your machine.

:::
```